website/docs/hoodie_deltastreamer.md

```diff
@@ -340,6 +388,26 @@ to trigger/processing of new or changed data as soon as it is available on S3.
 
 Insert code sample from this blog: https://hudi.apache.org/blog/2021/08/23/s3-events-source/#configuration-and-setup
 
+### GCS Events
+Google Cloud Storage (GCS) service provides an event notification mechanism which will post notifications when certain
+events happen in your GCS bucket. You can read more at [Pub/Sub Notifications](https://cloud.google.com/storage/docs/pubsub-notifications/).
+GCS will put these events in a Cloud Pub/Sub topic. Apache Hudi provides a GcsEventsSource that can read from Cloud Pub/Sub
+to trigger/processing of new or changed data as soon as it is available on GCS.
+
+#### Setup
+A detailed guide on [How to use the system](https://docs.google.com/document/d/1VfvtdvhXw6oEHPgZ_4Be2rkPxIzE0kBCNUiVDsXnSAA/edit#heading=h.tpmqk5oj0crt) is available.
```

Comment:
we usually don't link to adhoc google docs. it should be part of the RFC. curious to know why not update the RFC only ?

Reply:
This is the only publicly available reference doc for this feature. It was contributed to OSS (by me) a while after being developed. There's an older version of this doc but it is not public.

Reply:
I think we need not put the whole document. We typically assume that users know how to enable event notifications. What we can add here is the two spark-submit command samples for the two sources.

Reply:
Have added the spark-submit commands.